### PR TITLE
replaced array name with element in expect for string input

### PR DESCRIPTION
**Pull request by stacywile**

### CODE REVIEW

**Comment by ryanbmilbourne** [comment]
@stacywile can you elaborate further on your change?  What issue did you encounter that necessitated the change?

**Comment by gcochard** [comment]
Reviewing this, it seems to be fixing an issue where `expectation` was used in place of the actual variable name `exp`.

**Comment by gcochard** [comment]
This change should also be applied to lines 300-302, as they are still using the old, invalid `expectation` name.

**Comment by ryanbmilbourne** [comment]
Please also roll the patch version and an entry in `CHANGELOG.md`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+v0.5.0 - January 28 2016
```
Comment (ryanbmilbourne) [inline CHANGELOG.md]: Should be `v0.5.1`.  Looks like an entry for 0.5.0 was missing.

**Comment by ryanbmilbourne** [comment]
@stacywile Please address the version mismatch in the changelog and squash your commits, and then we can merge. :+1:

**Comment by gcochard** [comment]
Looks like travis is broken :cry: @stacywile please do a git commit --amend, change the text slightly, then force push your branch.

**Comment by ryanbmilbourne** [comment]
:+1: Yay travis came back.  :rocket: :ship: :shipit: :100: Thanks for contributing!

**Pull request merged**